src/main/java/randoop/main/GenTests.java

```diff
@@ -510,7 +517,69 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
-    }
+
+      Set<String> flakyTestNames = codeWriter.getFlakyTestNames();
```

Comment:
Can you abstract this out into a method?
Method handle() is already obscenely long, but let's not make it worse.